crates/weaver_forge/templates/test/weaver.yaml

```diff
@@ -27,10 +27,16 @@ templates:
     filter: semconv_grouped_attributes
     application_mode: single
   - pattern: "**/event.md"
-    filter: semconv_signal("event"; {}) | semconv_group_signals_by_root_namespace("events")
+    filter: >
+      semconv_signal("event"; {})
```

Comment:
We don't provide `semconv_...` dedicated filters for events, resources, and spans. So in this generic unit test, I'm using a custom filter for internal purposes only.